gradio/interface.py

```diff
@@ -981,7 +984,10 @@ def __init__(
                 )
             with Tabs():
                 for interface, tab_name in zip(interface_list, tab_names, strict=False):
-                    with Tab(label=tab_name):
+                    with Tab(
+                        label=tab_name,
+                        scale=1 if isinstance(interface, ChatInterface) else 0,
```

Comment:
instead of checking if its an instance of ChatInterface, you should just check if its any Blocks whose fill_height attribute is True, right?